superset-installer/etc/reflex-provisioner/inventory/templates/group_vars/global/all/raf/superset.yml

```diff
@@ -125,4 +125,6 @@ RVF_HELP_ENABLED: False
 SUPERSET_SESSION_LIFETIME_SECONDS: "3600" # stale session timeout
 RVF_LOG_LEVEL: "INFO"
 RVF_CACHE_DEFAULT_TIMEOUT: "{{ 60 * 60 * 24 }}"
-RVF_CACHE_TYPE: 'null'
\ No newline at end of file
+RVF_CACHE_TYPE: 'null'
+RVF_ENABLE_CORS: True
```

Comment:
if we are adding new variables in inventory then we should add that variables and realated changes in below github also. https://github.com/Guavus/ansible-role-raf-viz-fwk/tree/feature/ansible-structure-UIC-815

Reply:
Thanks for notifying the same. I am already maintaining this list for additional changes that came post RAF-3.6 release. I will collate all the info and float soon